### PR TITLE
[#889] gke iam

### PR DESCRIPTION
**Pull request by alexander-semenets**

### CODE REVIEW

File: helms/legion/values.yaml [hunk 1]
```diff
@@ -239,10 +239,29 @@ feedback:
...
     # Where feedback data should be stored
-    # Only S3 is supported nowadays
     # Valid values:
     # s3 - AWS S3
-    target: s3
+    # gcs - Google Cloud Storage
+    target: "s3"
+
+    # Detail configuration for storing on GCS
+    gcs:
```
Comment (kirillmakhonin) [inline helms/legion/values.yaml:248]: Can we add plain authorization (by providing JSON in helm values) to allow users use our fluentd without `k8s-gke-service-account-assigner` (as we did with AWS)?

File: containers/fluentd/Dockerfile [hunk 2]
```diff
@@ -18,3 +18,6 @@ FROM k8s.gcr.io/fluentd-elasticsearch:v2.0.4
 
 # Install fluentd-s3 plugin
 RUN gem install -N fluent-plugin-s3 -v '~>1.1.4'
```
Comment (kirillmakhonin) [inline containers/fluentd/Dockerfile]: Maybe we should add `--no-document` to `fluent-plugin-s3` too?

**Review by kirillmakhonin — CHANGES_REQUESTED** [review]

File: helms/legion/values.yaml (hunk 1 above)
Comment (karbyshevds) [reply helms/legion/values.yaml:248]: done in e6a96cf
Add keyfile based auth  for Google Cloud:
- `keyfile` authorization option
- keyfile values should be provided in values.yaml
- chart creates configmap based on provided values and mounts it to Fluentd container

File: containers/fluentd/Dockerfile (hunk 2 above)
Comment (karbyshevds) [reply containers/fluentd/Dockerfile]: done in 477a07a

**Comment by alexander-semenets** [comment]
bot, test app

**Comment by karbyshevds** [comment]
bot, test app

**Comment by karbyshevds** [comment]
bot, test app

**Comment by karbyshevds** [comment]
bot, test app

**Comment by mcdoker18** [comment]
Changes were merged to the #917  PR.

**Pull request closed without merging**